projects/storefrontlib/src/lib/product/components/product-list/product-list-item/product-list-item.component.scss

```diff
@@ -1,73 +1,51 @@
 @import 'theme';
-@import 'functions';
-@import 'mixins';
-
-$cx-search-list-border: 1px solid;
-$cx-search-list-border-color: 'light' !default;
-$cx-search-list-trbl-margin: 0 0 25px 0;
-
-$cx-search-list-image-width: 100% !default;
-
-$cx-search-list-name-trbl-margin: 0 0 3px 0 !default;
-$cx-search-list-name-color: 'dark' !default;
-$cx-search-list-name-hover-color: 'primary' !default;
-
-$cx-search-list-price-trbl-margin: 0 0 25px 0 !default;
-
-$cx-search-list-price-new-trbl-margin: 0 0 0 5px !default;
-
-$cx-search-list-price-old-color: 'secondary' !default;
-$cx-search-list-price-new-color: 'primary' !default;
 
 .cx-product-search-list {
-  border-bottom: $cx-search-list-border;
-  @include var-color('border-color', $cx-search-list-border-color);
-  margin: $cx-search-list-trbl-margin;
+  border-width: var(--cx-border-width, 0 0 1px 0);
+  border-style: var(--cx-border-style, solid);
+  border-color: var(--cx-border-color, var(--cx-g-color-light));
+  margin: var(--cx-margin, 0 0 25px 0);
 
   &:last-of-type {
-    border-bottom: none;
-  }
-
-  &__image__container {
-    display: block;
-    text-align: center;
+    border-width: var(--cx-border-width, 0 0 0 0);
   }
+}
 
-  &__image {
-    width: $cx-search-list-image-width;
-  }
+.cx-product-image-container {
+  display: var(--cx-display, block);
+  text-align: var(--cx-text-align, center);
+}
 
-  &__name {
-    @include type('5');
-    text-align: left;
-    display: block;
-    margin: $cx-search-list-name-trbl-margin;
-    @include var-color('color', $cx-search-list-name-color);
-    text-decoration: none;
+.cx-product-image {
+  width: var(--cx-width, 100%);
+}
 
-    &:hover {
-      @include var-color('color', $cx-search-list-name-hover-color);
-    }
-  }
+.cx-product-name {
+  @include type('5');
+  text-align: var(--cx-text-align, left);
+  display: var(--cx-display, block);
+  margin: var(--cx-margin, 0 0 3px 0);
+  color: var(--cx-g-color-dark);
+  text-decoration: none;
 
-  &__price--container {
-    text-align: left;
+  &:hover {
+    color: var(--cx-g-color-primary);
   }
+}
 
-  &__price {
-    @include type('3');
-    text-align: left;
-    display: inline-block;
-    margin: $cx-search-list-price-trbl-margin;
+.cx-product-price {
+  @include type('3');
+  text-align: var(--cx-text-align, left);
```

Comment:
I don't think this you be a variable you should just remove it if it's the default. In .cx-product-name also.